saleor/graphql/app/types.py

```diff
@@ -2,15 +2,87 @@
 from graphene_federation import key
 
 from ...app import models
+from ...core.exceptions import PermissionDenied
 from ...core.permissions import AppPermission
 from ..core.connection import CountableDjangoObjectType
 from ..core.types import Permission
 from ..core.types.common import Job
 from ..meta.types import ObjectWithMetadata
-from ..utils import format_permissions_for_display
+from ..utils import format_permissions_for_display, get_user_or_app_from_context
 from ..webhook.types import Webhook
-from .enums import AppTypeEnum
-from .resolvers import resolve_access_token
+from .dataloaders import AppByIdLoader, AppExtensionByAppIdLoader
+from .enums import (
+    AppExtensionTargetEnum,
+    AppExtensionTypeEnum,
+    AppExtensionViewEnum,
+    AppTypeEnum,
+)
+from .resolvers import (
+    resolve_access_token_for_app,
+    resolve_access_token_for_app_extension,
+)
+
+
+class AppManifestExtension(graphene.ObjectType):
+    permissions = graphene.List(
+        graphene.NonNull(Permission),
+        description="List of the app extension's permissions.",
+        required=True,
+    )
+    label = graphene.String(
+        description="Label of the extension show on dashboard side.", required=True
+    )
+    url = graphene.String(
+        description="URL where iframe of extension is placed.", required=True
```

Comment:
Is it the URL of a view?
```suggestion
        description="URL of a view where extension's iframe is placed.", required=True
```